src/whatsnew/2.1.rst

```diff
@@ -53,7 +53,7 @@ Upgrade Notes
   ``stale=update_after`` behaviour is equivalent to ``stable=true&update=lazy``.
   The deprecated ``stale`` parameter will be removed in CouchDB 3.0.
 
-* The new :config:option:`httpd/max_http_request_size` configuration parameter
+* The new :config:option:`chttpd/max_http_request_size` configuration parameter
```

Comment:
Let's not modify 2.1 release

Reply:
How about changing it to an inline literal?